src/python/WMCore/MicroService/Unified/MSTransferor.py

```diff
@@ -381,10 +381,9 @@ def makeTransferRequest(self, wflow):
                     if aboveWarningThreshold:
                         emailSubject = "[MS] Large pending data transfer under request id: {transferid}".format(
                             transferid=transferId)
-                        emailMsg = """Data transfer of size: {dsize} TBs has been suscribed
-                                      for {dtype} dataset: {dset}.""".format(dsize=teraBytes(dataSize),
-                                                                             dtype=dataIn['type'],
-                                                                             dset=dataIn['name'])
+                        emailMsg = "Workflow: {}\nhas a large amount of ".format(wflow.getName())
+                        emailMsg += "data subscribed: {} TB,\n".format(teraBytes(dataSize))
+                        emailMsg += "for {} data: {}.""".format(dataIn['type'], dataIn['name'])
```

Comment:
I know it may look like redundant information, but it would be good to have the transfer Id not only in the message subject but also in the message body too. Sometime people (especially judging on myself) tend to overlook the small but important details from the subject and rely only on the information provided int the message body.